mockld/streaming_service.go

```diff
@@ -24,9 +24,7 @@ func (l eventSourceDebugLogger) Printf(fmt string, args ...interface{}) {
 }
 
 type StreamingService struct {
-	URL         string
 	sdkKind     SDKKind
-	credential  string
```

Comment:
I removed SDK key validation from this low-level component; the mock endpoints will now accept the request no matter what. Instead, there's now a specific test in `server_side_stream_request.go` that looks at the Authorization header after the client has connected.